tests/test_json_connector.py

```diff
@@ -0,0 +1,27 @@
+from nexus_constructor.json_connector import JsonConnector
+from nexus_constructor.qml_models.instrument_model import InstrumentModel as model
+
+
+def test_good_json_returns_true():
+
+    json_connector = JsonConnector()
+
+    with open("tests/sample.json", mode="r") as file:
```

Comment:
It is good practice to avoid disk and network io in units tests, because we want to make them fast and independent of external systems (filesystem, network).

Instead of using files here you can simply create strings in the test. This has the advantage of the input being immediately visible to someone reading the test implementation. Creating it as a "raw string" using `r` in front of the quotes (`r"some string"`) will avoid needing to escape new line characters etc. Try to give shorter examples of the "good" and "bad" inputs.